doc/source/overview.rst

```diff
@@ -10,6 +10,12 @@ This `SciKit <http://scikits.appspot.com>`_ is developed by the SciPy
 community.  All contributions are most welcome!  Please join us on the
 mailing list (address provided below).
 
+Read about our `governance structure <skips/0-governance.html>`_ and our
+`mission, vision, and values <values.html>`_.
+
+Major proposals to the project are documented in
```

Comment:
I suggest to use different abbreviation, more consistent with the naming schemes of Python (Python Enhancement Proposal, PEP), NumPy (Numpy Enhancement Proposal, NEP), scikit-learn (Scikit-Learn Enhancement Proposal, SLEP) - SIEP (Scikit-Image Enhancement Proposal), or SKIEP (SciKit-Image ...) if you do not like the former.

SKIP sounds like something one should rather avoid doing :).

Reply:
changing the nomenclature of SKIP requires a SKIP. =P =P =P

More seriously, we've discussed this extensively in the past, and decided on SKIP as being suggestive of a fast move to the future — which is what it is! =) imho all the other options are terrible, and so is SLEP, which makes me think of the sound I might make when falling flat on my face in the mud.

Reply:
Ha-ha, I have the same feeling about SLEP 🤣.

Alright, let me try to collect my counter-arguments and I will get back with a solid S[K]I[E]P. :)